presto-cache/src/main/java/com/facebook/presto/cache/alluxio/PrestoCacheContext.java

```diff
@@ -50,7 +51,7 @@ private PrestoCacheContext(HiveFileContext hiveFileContext)
     @Override
     public void incrementCounter(String name, long value)
     {
-        hiveFileContext.incrementCounter(name, value);
+        hiveFileContext.incrementCounter(name, RuntimeUnit.NONE, value);
```

Comment:
static import `NONE`

Reply:
> Thanks for taking this followup task!! One thing worth noting is that in Presto codebase, we do static import when possible.

I see, will keep that in mind thanks. Just curious I saw some cases where the class is just imported and constants are used from there. When would we do that instead?

Reply:
done